ramls/patron.raml

```diff
@@ -61,6 +61,11 @@ traits:
             required: false
             type: boolean
             default: false
+          sortBy:
+            description: |
+              Indicates the order of records within the lists of holds, charges, loans
+            required: false
+            type: string
```

Comment:
How do clients of the module know what options they can provide to this parameter?

Reply:
I think via experimentation?  It may be worth updating the readme or API docs (via RAML) to indicate some of the more common fields.  I'm not sure being exhaustive is necessary.

Reply:
I little updated description and added example of CQL query for sortBy

Reply:
> > At the moment, many modules effectively ignore the sort in that case.
> 
> Are you aware of any that don't behave this way?

Not off the top of my head. I don't think we have any standard for this.

> > In the future, modules might disallow sorting on properties that do not exist.
> 
> Where can I find more information on this?

This has been an on and off discussion of aspects like this e.g. https://issues.folio.org/browse/RMB-534

Reply:
> I think via experimentation? It may be worth updating the readme or API docs (via RAML) to indicate some of the more common fields. I'm not sure being exhaustive is necessary.

I think some documentation would be good. Given that sort options will work differently with each record type, I think discovering options for this could be challenging.

Reply:
but once you find one that works, it'll be oh sooo satisfying ;)

Reply:
@marcjohnson-kint I updated README.md with detailed information regarding sortBy parameter and examples of requests

Reply:
@illia-borysenko Thank you for adding some documentation to the readme